run.py

```diff
@@ -119,18 +117,22 @@ def harvest_by_null(queue='priority-bumblebee',
             last_id = kv.value
         else:
             last_id = -1 
-                
-    
+    app.logger.info('harvest_by_null running with last_id of {}'.format(last_id))
+
     while True:
-        r = client.get(url, params=params)
+        # reuse the http ads api client with keep-alive connections and bearer token
+        r = app._client.get(url, params=params)
+        import pdb
+        pdb.set_trace()
         r.raise_for_status()
         j = i
         for d in r.json():
-            msg = TurboBeeMsg(target=d['target'],
-                              qid=d['qid'])
-            
+            if iscachable(d['target']):
+                msg = TurboBeeMsg(target=d['target'],
+                                  qid=d['qid'])
             
-            tasks.task_priority_queue.delay(msg)
+                app.logger.info('harvest_by_null queuing target {}'.format(d['target']))
+                tasks.task_priority_queue.delay(msg)
```

Comment:
here i'd recommend using logging to warn the user something, that was explicitly requested, was rejected and won't be processed